### ISSUE TITLE
scripts/postinstall.sh not found

### ISSUE THREAD

**Issue by NazarYablonskiy** [wix/detox#627]
Hi,
I'm getting following
(CI, Linux, detox@7.2.0)

```
...
error An unexpected error occurred: "/builds/xxx/xxx/xxx/mobile/node_modules/detox: Command failed.
  Exit code: 127
  Command: sh
  Arguments: -c scripts/postinstall.sh
  Directory: /builds/xxx/xxx/xxx/mobile/node_modules/detox
  Output:
  sh: scripts/postinstall.sh: not found".
...
```

Let me know if you need more details.
Thanks!

**Comment by rotemmiz** [comment]
Where is this output from ? 
Please be more descriptive in the first place, and follow the template.

**Comment by Stanok3694** [comment]
Hi. I had the same issue now, when I tried to install detox into my RN 0.50.3 project...

PS C:\Aigh\hb-aigh-main-fe> npm i detox --save-dev

> detox@7.3.4 postinstall C:\Aigh\hb-aigh-main-fe\node_modules\detox
> scripts/postinstall.sh

and after this I have got following error:
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! detox@7.3.4 postinstall: `scripts/postinstall.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the detox@7.3.4 postinstall script.

What does it mean?
